Formula/lensfun@0.3.95.rb

```diff
@@ -0,0 +1,52 @@
+class LensfunAT0395 < Formula
```

Comment:
If this is an alpha version, then we should not include it in `homebrew-core`.

https://docs.brew.sh/Versions#acceptable-versioned-formulae
> - Unstable versions (alpha, beta, development versions) are not acceptable for versioned (or unversioned) formulae.

Reply:
Thanks @cho-m. Removed accordingly `lensfun@0.3.95.rb` and added `head` to `lensfun.rb`.